### PR TITLE
Monitoring: Allocations active metric

### PR DESCRIPTION
**Pull request by amm0nite**
Hello!

I'm currently monitoring a TURN server using Stunner (Headless deployment model) and have come across an issue with the stunner_listener_connections metric. This metric is described in the documentation as representing the "Number of active downstream connections at a listener." However, in my setup, it consistently reports a value of 16, which coincidentally matches the --udp-thread-num setting, regardless of the service's actual usage.

I found an [answer in the discord](https://discord.com/channels/945255818494902282/1173611480080449647/1173738162502258768) that explains that it is expected behavior when using UDP for Stunner as I do. And then there is an interesting message:

> That being said, we should really implement a TURN allocation counter that would wrap this API: [stunner package - github.com/l7mp/stunner - Go Packages](https://pkg.go.dev/github.com/l7mp/stunner#Stunner.GetActiveConnections.)  This is a TODO list item currently I'm afraid. If you feel you really need this, please submit an issue at GitHub, we tend to prioritize wishlist items

I looked at the code and found that it was almost already implemented. I made the changes to make it work and validated in my test environment. As suggested in the discord message, i want to submit this to voice the need for the metric!

### CODE REVIEW

**Comment by levaitamas** [comment]
Hi @amm0nite, 

Thanks for the PR. looks fantastic! I will wait for @rg0now 's response before merging it.
If it is not a secret, mind to tell us a few words about your use-case?

**Review by rg0now — APPROVED** [review]

**Comment by rg0now** [comment]
Thanks a lot, this is really useful. If you have similar PRs, don't hold them back...:-)

**Comment by amm0nite** [comment]
Thanks for checking and merging my pull request!
We use Stunner to help with WebRTC video streaming.
With this metric I can keep an eye on usage stats and spot if there's a sudden dip in service utilization. This could help me figure out quickly if users are having trouble accessing the service.

**Pull request merged**
